### ISSUE TITLE
IE11 bugfix causes issues with tab order

### ISSUE THREAD

**Issue by sb-relaxt-at** [selectize/selectize.js#877]
In IE 11.0 tabbing through Selectize fields is broken, try [this example](http://jsfiddle.net/maLtpveo/4/), focus the first field and then press tab. It should now focus the second selectize (note that there are even added tabindex properties).

The reason seems to be the bugfix for "IE11 bug: element still marked as active" in [onBlur](https://github.com/brianreavis/selectize.js/blob/master/src/selectize.js#L625):

```
(dest || document.body).focus();
```

This causes the document's body to be focused. Removing this line resolves the issue (see [this fiddle](http://jsfiddle.net/maLtpveo/6/)), but I am not aware about the reason why this line has been added (and what issue it is trying to fix).

**Comment by jods4** [comment]
This "bugfix" does indeed create a bug which prevent proper keyboard navigation in IE11.
Removing the lines makes tab navigation work.

BUT it breaks "activate/focus" management in IE11. I noticed without this line that if you click in another textbox when selectize has focus, although the textbox seems focused the proper "activate/focus" events are not raised, which can cause problems, e.g. if your target textbox hosts a JS date picker.

So I'm looking for something that would fix both issues!

**Comment by jods4** [comment]
#406 seems to be the reason why this line of code is here.

**Comment by jods4** [comment]
OK I fixed it by changing this line:

```
(dest || document.body).focus();
```

to:

```
dest && dest.focus();
```

i.e. I'm only focusing the next control if I know one (which fixes the "no activate/focus on click" issue and let it go otherwise (which fixes tab navigation).

As far as I can tell it works nicely. Because I don't fully grasp what was creating the original problem (broken keyboard navigation), I can't say I'm 100% confident this change doesn't have adverse side-effects.

**Comment by sb-relaxt-at** [comment]
Thanks for your efforts. I will try this modified solution in our current usage scenario.

**Comment by brownbl1** [comment]
I've also tried this proposed fix and it has solved our IE bug as well, FWIW. I haven't done thorough testing though.

**Comment by bwilson-ux** [comment]
Perfect resolution - current project I am working on requires numerous instances of the selectize menu. This proposed fix works very well. :+1:

**Comment by sb-relaxt-at** [comment]
Just realized that I had patched it that way back in August. From my point of view it is working fine too.

**Comment by joallard** [comment]
Closed by #997, thanks all
